### DISCUSSION TITLE
Allow errors in app.transaction

### DISCUSSION THREAD

**Discussion by neokry** [wevm/frog-archived#78] [Ideas]
Currently `app.transaction` requires a `TransactionResponse` return value. If a request fails validation ie if input text is below a certain value or if a query string is invalid an error response should be returned instead of a broken `TransactionResponse`.

**Comment by jxom** [comment]
You can now return `Response` objects [from `0.3.1`](https://github.com/wevm/frog/releases/tag/frog%400.3.1):

```ts
app.transaction('/tx', c => {
  return new Response('dang', { status: 400 })
})
```
